source/manage-apps/deploy/deployment-models-and-regions.txt

```diff
@@ -70,7 +70,15 @@ Cloud Deployment Regions
 - Sydney (``ap-southeast-2``)
 - Virginia (``us-east-1``)
 
-The following regions are available for :ref:`local deployment
+{+service+} supports deployment and writes in the following Azure regions:
+
+- ``EUROPE_WEST``
```

Comment:
Do you want this to align to what we show on the UI?

![Screen Shot 2022-06-06 at 2 41 37 PM](https://user-images.githubusercontent.com/13564380/172235582-b443f773-502c-4401-b59b-b439047e2360.png)

Reply:
Updated to match this list, thank you!